### ISSUE TITLE
Gemdog Start Here

### ISSUE THREAD

**Issue by nmsanchez** [nmsanchez/noonchez#1]
Hey @gemdog: This is where we start discussing stuff. Go ahead and add a comment to this issue.

**Comment by gemdog** [comment]
nub is defective

**Comment by gemdog** [comment]
i love you

**Comment by nmsanchez** [comment]
Wonderful, @gemdog. Now put a picture of Nub in this issue.

**Comment by gemdog** [comment]
![screen shot 2015-08-31 at 8 21 44 pm](https://cloud.githubusercontent.com/assets/13491611/9979176/a5e098ba-5f12-11e5-8f61-912605406332.png)

**Comment by gemdog** [comment]
@nmsanchez i love you. thank you for birthing me

**Comment by nmsanchez** [comment]
You are welcome, @gemdog.

Now click on the "Markdown supported" link up there ^ and read a bit about markdown. In your next comment, tell me something interesting 

### in bold
and 

*in italics*

**Comment by gemdog** [comment]
<h1>  **Learning Code** </h1>

<h2> *Is Fun* </h2>

**Comment by gemdog** [comment]
<h1> *Is Fun*</h1>

**Comment by gemdog** [comment]
<h1> _Trying this again._</h1>

**Comment by gemdog** [comment]
_Hm, having some trouble with this_

**Comment by nmsanchez** [comment]
You can always click "Preview" before you post. It's good to see how it will look before you click "Comment."

Now try to copy and paste a short poem. Post it all in italics. :clap: :clap: :clap:

**Comment by gemdog** [comment]
_Dayman (ah-ah-ah)
Fighter of the Nightman (ah-ah-ah)

Champion of the Sun (ah-ah-ah)
You're a Master of Karate
And friendship
For Everyone

Dayman (ah-ah-ah)
Fighter of the Nightman (ah-ah-ah)
Champion of the Sun! (ah-ah-ah)
You're a Master of Karate

And Friendship
For Everyone_

**Comment by nmsanchez** [comment]
[Do you know how to make words link to a web page](http://www.fxnetworks.com/shows/its-always-sunny-in-philadelphia/episodes)?

Click the blue words to see where it takes you.

Then click on the edit pencil above and to the right to see how I did this. Then *you* post a comment with a link to something else.

**Comment by gemdog** [comment]
[i love dogs](https://www.youtube.com/watch?v=mKZBjJwUQAY)

cool!

**Comment by nmsanchez** [comment]
Nooooooooooo!

**Comment by gemdog** [comment]
sorry did i do it wrng

**Comment by nmsanchez** [comment]
:star2: !

Now it's time for you to open a new Issue in this repository. 

- [ ] Open a new issue
- [ ] Name it "Inviting Collaborators"
- [ ] @- mention @bescalante + @agelender in your new issue. I've added them as collaborators.

**Comment by nmsanchez** [comment]
>sorry did i do it wrng

No, it was great. I was just saying "no" to the song.

**Comment by gemdog** [comment]
Inviting Collaborators

@bescalante @agelender

**Comment by nmsanchez** [comment]
Nope, Go up to the top and on the top right, click "New Issue." It's a green button.
